README.md

```diff
@@ -4,58 +4,66 @@ This is a lightweight image writing app with a simple four-step GUI for creating
 
 ## Translation Changes
 
-Changes to master language (en_GB) strings can be submitted via pull request to this GitHub repo. English strings are periodically synchronised to a Transifex project: https://www.transifex.com/libreelec/libreelec-usb-sd-creator allowing contributors to translate them into other languages. Translated strings are periodically synchronised from Transifex back to this repo for inclusion in the next release.
+Changes to master language (en_GB) strings can be submitted via pull request to this GitHub repo. English strings are periodically synchronised to a Transifex project: <https://www.transifex.com/libreelec/libreelec-usb-sd-creator> allowing contributors to translate them into other languages. Translated strings are periodically synchronised from Transifex back to this repo for inclusion in the next release.
 
 Pull requests for all other languages will be rejected. You will be asked to submit changes via Transifex.
 
 ## Translation Languages
 
-Requests for new languages should be made on Transifex. Once a new language has been approved it will be added to the translation project where you can conribute translated strings.
+Requests for new languages should be made on Transifex. Once a new language has been approved it will be added to the translation project where you can contribute translated strings.
 
 Please note: languages are translated by people (you) not magic!
 
 ## Issues and Support
 
-Issues should be reported via the forum here: https://forum.libreelec.tv/board/41-usb-sd-creator-support
+Issues should be reported via the forum here: <https://forum.libreelec.tv/board/41-usb-sd-creator-support>
```

Comment:
not clear why the angle brackets are needed

Reply:
https://github.com/LibreELEC/usb-sd-creator/pull/66#discussion_r1504874401